scripts/export.py

```diff
@@ -119,6 +122,26 @@ def load_controls(loaded_rules: dict):
             logging.error(f"failed to open control: '{path_in_str}'")
             raise TypeError(e)
         
+        categoryFieldName = "category"
+        subCategoryFieldName = "subCategory"
+        
+        # insert category ID into control obj
+        if categoryFieldName in new_control:
```

Comment:
Didn't we agree to validate that a control has categories?

Reply:
no, some controls are missing categories definitions on the document. For now, there is no reason to enforce it in every control